<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseDetails</class>
 <widget class="QWidget" name="CourseDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>55</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CourseDetails</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#eddedf</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="details_layout">
    <item>
     <layout class="QHBoxLayout" name="course_id_layout">
      <item>
       <widget class="QLabel" name="course_id_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>华文新魏</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>课程号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="course_id">
        <property name="font">
         <font>
          <family>华文新魏</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="teacher_name_layout">
      <item>
       <widget class="QLabel" name="teacher_name_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>华文新魏</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>任课教师：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="teacher_name">
        <property name="font">
         <font>
          <family>华文新魏</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
